src/Sarif.FunctionalTests/UpdateBaselines.ps1

```diff
@@ -38,13 +48,13 @@ function Build-Baselines($toolName)
 		
         # Actually run the converter
         Remove-Item $outputTemp -ErrorAction SilentlyContinue
-        & Write-Host "$utility convert --input ""$input"" --tool $toolName --output ""$outputTemp"" "
-        &$utility convert --input "$input" --tool $toolName --output "$outputTemp" --pretty --persist-file-contents
+        & Write-Host "$utility convert ""$input"" --tool $toolName --output ""$outputTemp"" -p --persist-file-contents"
+        &$utility convert "$input" --tool $toolName --output "$outputTemp" -p --persist-file-contents
```

Comment:
Maybe _this_ has something to do with the reason the file contents went away in the PREfast examples.